.all-contributorsrc

```diff
@@ -14,6 +14,7 @@
         "code"
       ]
     },
+ 
```

Comment:
@Arpaninformationtech there is this extra line over here. Just remove this too.

Reply:
ok done @arnabsen1729